pkg/visor/duration.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 // Duration wraps around time.Duration to allow parsing from and to JSON
+// TODO: Put this in a suitable module.
```

Comment:
Do we need our custom type at all?

Reply:
@nkryuchkov I think it makes the config a bit cleaner. Other we will need to work with nanoseconds.